smartseq2_single_sample/run_multi_samples/scripts/calculate_cost_workflow.py

```diff
@@ -0,0 +1,255 @@
+import argparse
+import json
+import math
+import urllib.request, urllib.error, urllib.parse
+import base64
+import dateutil.parser
+from io import StringIO
+import gzip
+import sys, re
```

Comment:
Same as above, `sys` and `re` do not appear to be used in the script so they can be removed